docs/static/upgrading.asciidoc

```diff
@@ -68,21 +68,33 @@ If you are installing Logstash with other components in the Elastic Stack, also
 
 ==== Issue with Logstash 6.0.0 `document_type` when writing to Elasticsearch 6.x
 
-We’d like to alert users to behavior in Logstash 6.0.0 that can cause errors writing to Elasticsearch 6.0+ clusters. When the `type` field has different values for different events per Elasticsearch index Logstash will encounter errors while indexing documents. These errors look something like the example below, which has been shortened from the full message. This bug is likely to be encountered by users using either Beats, or multiple inputs with different `type` values.
+We’d like to alert users to behavior in Logstash 6.0.0 that can cause errors when writing to Elasticsearch 6.0+ clusters. When Logstash attempts to index events that result in multiple `type` values, Logstash encounters indexing errors. These errors look similar to the following example, which has been shortened from the full message:
 
-`[2017-11-21T14:26:01,991][WARN ][logstash.outputs.elasticsearch] Could not index event to Elasticsearch. {:status=>400,  :response=>{"error"=>{"reason"=>"Rejecting mapping update to [myindex] as the final mapping would have more than 1 type: [type1, type2]"}}}}`
+[source,shell]
+----
+[2017-11-21T14:26:01,991][WARN ][logstash.outputs.elasticsearch] Could not index
+event to Elasticsearch.{:status=>400,  :response=>{"error"=>{"reason"=>"Rejecting
+mapping update to [myindex] as the final mapping would have more than 1 type:
+[type1, type2]"}}}}
+----
 
-The workaround for Logstash 6.0.0 is to add the setting `document_type => doc` to the Elasticsearch output configuration. We will issue a patch to address this issue soon in a new version of Logstash. 
+Users are likely to encounter this error when Logstash is receiving data from:
 
-Logstash has historically used the value of the `type` field to set the Elasticsearch `type` by default. Elasticsearch 6 [no longer supports more than one type]((https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html) per index. This is why the new behavior will only be applied to Elasticsearch 6.0 + clusters with our upcoming fixes.
+* multiple types of Beats
```

Comment:
I realize this takes up a bit more space, but I think it's worth listing the situations where users might encounter the problem so that they can be proactive about solving it. The previous explanation seemed a bit too terse.